bazel/README.md

```diff
@@ -916,7 +916,27 @@ Edit the paths shown here to reflect the installation locations on your system:
 export CLANG_FORMAT="$HOME/ext/clang+llvm-12.0.1-x86_64-linux-gnu-ubuntu-16.04/bin/clang-format"
 export BUILDIFIER_BIN="/usr/bin/buildifier"
 ```
-
+The easiest way to use the correct `clang-format` in your host system is to copy the `clang-format` from the ci docker image.
+* Run the ci docker image
+```shell
+ci/run_envoy_docker.sh bash
+```
+* Get the docker container ID
+```shell
+docker ps | grep envoy-build-ubuntu | awk '{print $1}'
```

Comment:
v minor nit (thanks for iterating so far!)

maybe set the `dockerContainerID` here like...

```
dockerContainerID=$(docker ps | grep envoy-build-ubuntu | awk '{print $1}')
```

it just means people can copypasta their way through